cli/graph_container.rs

```diff
@@ -94,6 +98,51 @@ impl MainModuleGraphContainer {
     self.check_specifiers(&specifiers).await
   }
 
+  fn resolve_files(
+    &self,
+    patterns: &[String],
+  ) -> Result<Vec<String>, AnyError> {
+    let mut resolved_files = Vec::new();
+
+    for pattern in patterns {
+      let expanded = glob::glob(pattern).expect("Failed to read glob pattern");
```

Comment:
Ideally this code should do something similar to what lint and fmt does (so not using glob directly and instead use `PathOrPatternSet`).

Reply:
ok

Reply:
done; please review it again.